website/blog/2018-05-23-1.13.0.md

```diff
@@ -242,9 +242,54 @@ declare class C mixins D {}
 
 These new features were already supported when using the Babylon parser, now Prettier 1.13 added support for them when using the Flow parser.
 
+### TypeScript
+
+#### Added support for TSImportType ([#4429] and [#4438] by [@ikatyang])
+
+A new feature added in TypeScript 2.9 for describing the shapes of a module imported dynamically.
+
+```ts
+// Input
+export const x:import(   "./foo"   )={x:0,y:0};
+
+export let y:import("./foo2"   ).Bar.I={a:"",b:0};
+
+export let shim:typeof import(   "./foo2")={Bar: Bar2};
+
+// Output with Prettier 1.13
+export const x: import("./foo") = { x: 0, y: 0 };
+
+export let y: import("./foo2") = { a: "", b: 0 };
```

Comment:
Looks like this is removing `.Bar.I`. Is this a bug?

Reply:
No it was done in 2 PRs and I used the wrong one to format the playground